### PR TITLE
Feature/minor changes

### PR DESCRIPTION
**Pull request by ZoranPavlovic**
- Add explicit error checking and logging of output for tesseract output parsing.
- Explicit mention of availability failure for specific failing version.
- Save library-loading errors in tesseract_raw
- Removed unnecessary global declarations.

### CODE REVIEW

File: src/pyocr/libtesseract/__init__.py [hunk 1]
```diff
@@ -212,11 +212,13 @@ def is_available():
     if not available:
         return False
     version = get_version()
+
     # C-API with Tesseract <= 3.02 segfaults sometimes
     # (seen with Debian stable + Paperwork)
     # not tested with 3.03
     if (version[0] < 3 or
             (version[0] == 3 and version[1] < 4)):
+        print("Unsupported version [%s]" % ".".join([str(r) for r in version]))
```
Comment (jflesch) [inline src/pyocr/libtesseract/__init__.py]: Please use the Python logging infrastructure:
```py
import logging
logger = logging.getLogger(__name__)
(...)
    logger.warning("Libtesseract: Unsupported version [%s] ...")
```

Comment (ZoranPavlovic) [reply src/pyocr/libtesseract/__init__.py]: Sure - Can do. Do you want me to add any debug logging while I'm at it?

Comment (jflesch) [reply src/pyocr/libtesseract/__init__.py]: It's good enough for this PR, thanks :)

**Comment by ZoranPavlovic** [comment]
@jflesch  - Thanks!

**Pull request merged**